lib/fusuma/plugin/appmatcher/gnome_extensions/installer.rb

```diff
@@ -13,6 +13,22 @@ class Installer
 
           EXTENSION = "./appmatcher@iberianpig.dev"
 
+          def gnome_shell_extension_path
+            output = `gnome-shell --version`
+            version = output.match(/GNOME Shell (\d+\.\d+)/)
+
+            if version
+              version_number = version[1].to_f
+              if version_number >= 45.0
+                return './appmatcher45@iberianpig.dev'
+              else
+                return './appmatcher@iberianpig.dev'
+              end
+            else
+              return './appmatcher@iberianpig.dev'
+            end
```

Comment:
The following directory structure may solve the problem. 

- ./gnome44/appmatcher@iberianpig.dev
- ./gnome45/appmatcher@iberianpig.dev

Reply:
fixed with https://github.com/iberianpig/fusuma-plugin-appmatcher/pull/8/commits/c301ebdf2f84446975862197d26dc3043ae9100e